config/simplesamlphp-users.php

```diff
@@ -19,36 +19,34 @@
               NOTE: we add a bunch of claims that we expect to get back from Seneca,
               and need to simulate here.
             */
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress' => 'user1@myseneca.ca',
+            'displayname' => 'Johannes Kepler', 
             'email' => 'user1@myseneca.ca',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname' => 'Johannes',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname' => 'Kepler',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name'=> 'user1@myseneca.ca',
-            'sAMAccountName'=> 'user1', 
-            'http://schemas.microsoft.com/identity/claims/displayname' => 'Johannes Kepler',  
+            'sAMAccountName'=> 'user1',
+            'group'=>'mycustomdomain-students',
         ),
         'user2:user2pass' => array(
             'uid' => array('2'),
             'eduPersonAffiliation' => array('group2'),
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress' => 'user2@myseneca.ca',
+            'displayname' => 'Galileo Galilei',
             'email' => 'user2@myseneca.ca',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname' => 'Galileo',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname' => 'Galilei',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name'=> 'user2@myseneca.ca',
             'sAMAccountName'=> 'user2',
-            'http://schemas.microsoft.com/identity/claims/displayname' => 'Galileo Galilei',
+            'group'=>'mycustomdomain-faculty',
         ),
+        'user3:user3pass' => array(
```

Comment:
Please add this user/password to the Markdown file where we keep track of all user accounts for testing/dev.